sql/src/main/java/io/crate/expression/symbol/Symbol.java

```diff
@@ -76,9 +76,12 @@ public String toString() {
      * The reasoning behind this is that we want to avoid
      * query Lucene performance to drop due to casts. This
      * is true for {@link Field}/{@link io.crate.metadata.Reference}.
+     *
+     * However, comparing two columns will never utilize the index so
```

Comment:
I think we should put this comment into the `Param` class where this is happening. Adding comments about implementation details that are elsewhere is bound to get outdated and confusing.